client/src/components/molecules/SignUpDetails/SignUpDetails.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Form, Col } from 'react-bootstrap';
+import DateOfBirthPicker from '../../atoms/DateOfBirthPicker/DateOfBirthPicker';
+
+export default function SignUpDetails() {
+  return (
+    <>
```

Comment:
why fragment?

Reply:
The form would not render without the fragment